examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -221,35 +229,79 @@ function rhs!(dydt, y, parameters, t, alpha, beta)
     cw = If2c.(w)
     cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
 
-    # The following, simple Flux-corrected Transport regression test (falling back to the 3rd-order upwinding, with C=1) is implemented following
-    # Ref: https://link.springer.com/book/10.1007/978-1-4419-6412-0 , Sec. 5.4 (p. 221)
-    corrected_antidiff_flux = similar(dρq1)
+    # 1) Horizontal transport for ρq1 (includes both horizontal and vertical velocity components)
     @. dρq1 = beta * dρq1 - alpha * hdiv(cuvw * ρq1) + alpha * ystar.ρq1
-    @. corrected_antidiff_flux = vdivf2c(
-        Ic2f(ρ) *
-        C *
-        (
-            third_order_upwind_c2f(w, ρq1 / ρ) -
-            first_order_upwind_c2f(w, ρq1 / ρ)
-        ),
-    )
+    # 2) Vertical transport for ρq1:
+    # 2.1) vertical advection by vertical velocity, corrected by BB FCT:
+    @. dρq1_td = -alpha * vdivf2c(Ic2f(ρ) * first_order_upwind_c2f(w, ρq1 / ρ))
     @. dρq1 -=
-        alpha * (
-            vdivf2c(Ic2f(ρ) * first_order_upwind_c2f(w, ρq1 / ρ)) +
-            corrected_antidiff_flux
+        dρq1_td -
+        alpha * vdivf2c(
+            FCTBB(
+                Ic2f(ρ) * (
+                    third_order_upwind_c2f(w, ρq1 / ρ) -
+                    first_order_upwind_c2f(w, ρq1 / ρ)
+                ),
+                dρq1_td / alpha,
```

Comment:
@simonbyrne I modified this as we discussed but this test still doesn't work. I also tried to lower the dt significantly, but didn't help. It must be something else that I am missing now...

Reply:
@simonbyrne , @charleskawczynski , @daniel my first thought (and question in the Slack channel) was that `dρq1_td` should account for the history of the vertical transport, according to the algorithm in the book. I tried to account for the history of this, by saving it as another state variable in [0fd9830](https://github.com/CliMA/ClimaCore.jl/pull/889/commits/0fd98304e01a4e2dd1421e409730d2e6b3ad1c2d), but it still didn't work. Can we please discuss these implementation details about how to use these FCT operators in 3D problems?